variables.tf

```diff
@@ -87,6 +87,12 @@ variable "db_admin_username" {
   default     = "postgres"
 }
 
+variable "db_name" {
+  description = "The name of the Orca database within the RDS cluster. Default set to disaster_recovery_PREFIX in main.tf."
```

Comment:
See comment on Name being PREFIX_name instead of name_PREFIX

Reply:
Fixed